Move citation info to separate file (#1)
CardiacModelling/WIRES: README.md

## Acknowledging this work

> If you publish any work based on the contents of this repository please cite it using the information given in the [CITATION](./CITATION) file.

Author: ```suggestion
To cite this work, please use the information given in the [CITATION](./CITATION) file.
```
